<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qVBF</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QToolButton" name="btn_open">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="qvbf.qrc">
          <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_save">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="qvbf.qrc">
          <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_export">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="qvbf.qrc">
          <normaloff>:/images/export.png</normaloff>:/images/export.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_import">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="qvbf.qrc">
          <normaloff>:/images/import.png</normaloff>:/images/import.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_add">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="qvbf.qrc">
          <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="btn_about">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="qvbf.qrc">
          <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="view"/>
      <widget class="QStackedWidget" name="stack">
       <widget class="QWidget" name="page_main"/>
       <widget class="QWidget" name="page_header">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>sw_part_number</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_part_number"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>sw_part_type:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_part_type"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>network</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_network"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>can_frame_format:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_can_frame_format"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>ecu_address:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sb_ecu_address">
             <property name="prefix">
              <string>0x</string>
             </property>
             <property name="maximum">
              <number>2147483647</number>
             </property>
             <property name="displayIntegerBase">
              <number>16</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>call:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sb_call">
             <property name="prefix">
              <string>0x</string>
             </property>
             <property name="maximum">
              <number>2147483647</number>
             </property>
             <property name="displayIntegerBase">
              <number>16</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QCheckBox" name="cb_erase">
             <property name="text">
              <string>Erase</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="text"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_block">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>addr:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="SpinBox" name="sb_block_addr">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="prefix">
              <string>0x</string>
             </property>
             <property name="decimals">
              <number>0</number>
             </property>
             <property name="maximum">
              <double>4294967295.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>size:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl_block_size">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="btn_block_open">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="qvbf.qrc">
               <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btn_block_save">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="qvbf.qrc">
               <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="wdg_hexview" name="hexview" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>wdg_hexview</class>
   <extends>QWidget</extends>
   <header>wdg_hexview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>spinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qvbf.qrc"/>
 </resources>
 <connections/>
</ui>
